File: 3723/3723.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var maximizeSumOfSquaresOfDigits = function(root) {
    if (!root) return 0;
    return 1 + Math.max(maximizeSumOfSquaresOfDigits(root.left), maximizeSumOfSquaresOfDigits(root.right));
};

File: 3723/3723.cs
public class Solution {
    public int MaximizeSumOfSquaresOfDigits(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MaximizeSumOfSquaresOfDigits(root.left), MaximizeSumOfSquaresOfDigits(root.right));
    }
}